fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/FluentIcon.kt

```diff
@@ -15,17 +15,23 @@ enum class IconType {
 
 data class IconSize(val size: Dp, val type: IconType)
 
-data class Icon(
-        val light: ImageVector = ImageVector.Builder("", 0.dp, 0.dp, 0F, 0F).build(),
-        val dark: ImageVector = ImageVector.Builder("", 0.dp, 0.dp, 0F, 0F).build(),
+data class FluentIcon(
+    val light: ImageVector = ImageVector.Builder("", 0.dp, 0.dp, 0F, 0F).build(),
+    val dark: ImageVector = light,
+    val contentDescription: String = "",
+    val onClick: (() -> Unit)? = null
 ) {
     @Composable
     fun value(themeMode: ThemeMode = com.microsoft.fluentui.theme.FluentTheme.themeMode): ImageVector {
         return when (themeMode) {
             ThemeMode.Light -> light
             ThemeMode.Dark -> dark
             ThemeMode.Auto -> if (isSystemInDarkTheme()) dark else light
-            else -> ImageVector.Builder("", 0.dp, 0.dp, 0F, 0F).build()
         }
     }
+
+    fun isAvailable(): Boolean {
```

Comment:
from the usage, it seems this function checks the existence of an icon. if such a case, can we rename it to something like "isIconAvailable"?